### ISSUE TITLE
Firefox Support

### ISSUE THREAD

**Issue by SunburnedGoose** [DestinyItemManager/DIM#255]

**Comment by kyleshay** [comment]
blocked by http://www.arewewebextensionsyet.com/

**Comment by TheHandsomeCoder** [comment]
Fixed by #456?

**Comment by kyleshay** [comment]
yup! we theoretically support FF- just have to push it out! lets track pushing to FF by re-using this issue number.

**Comment by bhollis** [comment]
Closing in favor of web strategy.
